mcp_hydrolix/mcp_server.py

```diff
@@ -137,7 +147,37 @@ def execute_query(query: str):
 
 @mcp.tool()
 def run_select_query(query: str):
-    """Run a SELECT query in a Hydrolix database"""
+    """Run a SELECT query in a Hydrolix time-series database using the Clickhouse SQL dialect.
+       Queries run using this tool will timeout after 30 seconds. Queries should include either a LIMIT
+       clause or a filter based on the primary key (which will be a timestamp) as a performance guard to
+       ensure they return in a reasonable amount of time. Queries should select specific fields and avoid
```

Comment:
I wonder if you reword this slightly, does it change the results?

> Queries should include either a LIMIT
       clause or a filter based on the primary key as a performance guard to
       ensure they return in a reasonable amount of time. The primary key used in the database is always a timestamp.

Reply:
It doesn't seem to, but it also doesn't hurt, so I've put left it in.